### ISSUE TITLE
apng图片有时候播放速度会变快

### ISSUE THREAD

**Issue by fuxuanyang11** [penfeizhou/APNG4Android#183]
### New Issue Checklist

- [ ] I have searched for a similar issue in the [project](https://github.com/penfeizhou/APNG4Android/issues) and found none

### Issue Info
重新加载几次apng图片可能会出现这种情况

| Info                         | Value                                                 |      |
| ---------------------------- | ----------------------------------------------------- | ---- |
| Device Info                  | 任何版本                                |      |
| System Version               | 任何版本                                             |      |
| APNG4Android Library Version | e.g.  2.24.0                                           |      |
| Repro rate                   | e.g. all the time (100%) / sometimes x% / only once   |      |
| Repro with our demo project  | e.g. does it happen with our demo project?            |      |
| Demo project link            | e.g. link to a demo project that highlights the issue |      |

### Issue Description and Steps

Please fill in the detailed description of the issue (full output of any stack trace, compiler error, ...) and the steps to reproduce the issue.

**Comment by fuxuanyang11** [comment]
这是文件地址：https://mfile.mokayy.com.cn/dress/b7ec2ec60a004a4ea3134318afff0897.png

**Comment by jingpeng** [comment]
重新加载几次apng是指，有具体代码么

**Comment by fuxuanyang11** [comment]
> 重新加载几次apng是指，有具体代码么

recyclerview 中imageview隐藏又重新显示几次后就会这样

**Comment by fuxuanyang11** [comment]
#176 有同样的问题
